packages/react-components/react-nav-preview/src/components/Nav/Nav.types.ts

```diff
@@ -36,6 +36,11 @@ export type NavProps = ComponentProps<NavSlots> & {
    * Mutually exclusive with defaultSelectedValue.
    */
   selectedValue?: NavItemValue;
+
+  /**
+   * Callback used by NavCategoryItem to request a change on it's own opened state
+   */
+  onNavCategoryItemToggle?: EventHandler<OnNavItemSelectData>;
```

Comment:
What would be the use case for this callback? I think if the goal is to know when an item is toggled, it might be good to know which item is toggled. What do you think?

Reply:
I think the use case would be if I wanted to pass the same function into all the children's category toggles. This would be useful if someone wanted to collect telemetry on the items.